src/metadata.rs

```diff
@@ -136,6 +136,11 @@ pub enum ItemKind {
         #[serde(skip)]
         item_handle: Handle<ItemMeta>,
     },
+    MeleeWeapon {
+        attack: AttackMeta,
+        audio: AudioMeta,
+        spritesheet: Box<FighterSpritesheetMeta>,
```

Comment:
Is there a reason we need to box here? (not saying its wrong, im just not super acquainted with Boxing)

Reply:
Clippy said it's best because of the size difference between the ItemKind Variants, I was surprised too

Reply:
Yeah, because an enum must take up as much memory as it's largest variant, and the `FighterSpritesheetMeta` struct makes the `MeleeWeapon` variant much larger than all the other variants, boxing it can make the enum take less space when using the non-`MeleeWeapon` variants.

I'm not sure if it's really a concern, but seems like we might as well trust clippy in this case.